parser.c

```diff
@@ -142,49 +158,52 @@ static int parse_readbuf_string(struct tinyiiod *iiod, char *str)
 		return -EINVAL;
 
 	tinyiiod_do_readbuf(iiod, device, (size_t) bytes_count);
+
```

Comment:
this line change looks un-necessary;
while i agree that it may look better with this newline, it's a good idea to add a separate commit